contrib/kde-cli-tools/template.py

```diff
@@ -0,0 +1,36 @@
+pkgname = "kde-cli-tools"
+pkgver = "6.0.5"
+pkgrel = 0
+build_style = "cmake"
+# FIXME: only test fails on initTestCase() 'fakeApplicationService' returned FALSE
+make_check_args = ["-E", "filetypestest"]
+make_check_env = {"QT_QPA_PLATFORM": "offscreen"}
+hostmakedepends = [
+    "cmake",
+    "extra-cmake-modules",
+    "gettext",
+    "ninja",
+]
+makedepends = [
+    "kcmutils-devel",
+    "kconfig-devel",
+    "kdoctools-devel",
+    "ki18n-devel",
+    "kiconthemes-devel",
+    "kio-devel",
+    "kparts-devel",
+    "qt6-qtbase-devel",
+    "qt6-qtsvg-devel",
+    # TODO: "kdesu-devel",
+]
+# depends = ["xdg-utils"]
+pkgdesc = "KDE command line tools to interact with Plasma"
+maintainer = "Jami Kettunen <jami.kettunen@protonmail.com>"
+license = "GPL-2.0-or-later AND LGPL-2.1-or-later"
+url = "https://invent.kde.org/plasma/kde-cli-tools"
+source = f"$(KDE_SITE)/plasma/{pkgver}/kde-cli-tools-{pkgver}.tar.xz"
+sha256 = "c0027ae04e691c692e1a8c0565d6779fb36ecb0d7af78f6663b9230f4581b28f"
+hardening = ["vis", "cfi"]
+
+
+# TODO: add kdesu (from ../libexec/kf6/kdesu) to default PATH with symlink?!
```

Comment:
do all the kde things that use kdesu find it without it being in path?

sounds fine to symlink it anyway